python/ray/scripts/scripts.py

```diff
@@ -725,6 +726,12 @@ def start(
 
     # Whether the original arguments include node_ip_address.
     include_node_ip_address = False
+
+    if node_ip_address is None and is_ipv6_address(node_ip_address):
```

Comment:
Since `node_ip_address` is `None`, `is_ipv6_address(None)` is always `False`, so the block never runs. Shouldn't it be:
```
if node_ip_address and is_ipv6_address(node_ip_address)
```